### DISCUSSION TITLE
Customize Sidebar Links

### DISCUSSION THREAD

**Discussion by jgodi** [storybookjs/storybook#13909] [Help] [answered]
I would like to customize the way the sidebar links work. I just want the top level component to be the link and to not have any of the links to the stories (we publish in docs only mode)

For example:

![CleanShot 2021-02-10 at 09 46 50](https://user-images.githubusercontent.com/870799/107549794-1578e900-6b85-11eb-866a-ae5fce357991.png)

I would just want links for Accordion, Accordion Summary, Actions, Details, etc.

**Comment by shilman** [comment, marked as answer]
You can run/build storybook with the `--docs` flag and it collapses the sidebar like you requested

https://storybook.js.org/docs/react/writing-docs/build-documentation

**Comment by AvielBitton** [comment]
![Screen Shot 2022-09-04 at 15 57 19](https://user-images.githubusercontent.com/87990236/188314663-35e73e7b-0816-4636-bd4c-522a29f9d90c.png)

in the manger-head.html
you can customize CSS issues, 

so, config the element you want to display 'none'

```

  a.sidebar-item:not([data-nodetype='document']) {
    display: none !important;
  }

```
